docs/assets/css/netlify.scss

```diff
@@ -30,7 +30,13 @@
   }
 }
 
-// Don't show the toggle code button in the Netlify CMS preview pane
+// Don't show the toggle code button in the Decap CMS preview pane
 .frame-content #toggle-code {
   display: none;
 }
+
+// By default the preview pane doesn't have any padding.
+.frame-content {
+  padding-left: 20px;
+  padding-right: 20px;
+}
```

Comment:
Can this entire file be deleted now that `decap.scss` exists? And import it in [`main.scss`](https://github.com/cfpb/design-system/blob/4dd8418fa6d637bb74de770fb3c8a4bbc4e74f1d/docs/assets/css/main.scss#L33).

Reply:
Mmm yes indeed https://github.com/cfpb/design-system/pull/1980/commits/d8423951840088e961ad9c0811b22ef033555236